test_apps/contracts_app/contracts/simple_storage.sol

```diff
@@ -2,14 +2,6 @@ pragma solidity ^0.4.17;
 
 import "ownable.sol";
 
-library Assert {
```

Comment:
Why did you get rid of this code?

Reply:
It was interfering with Assert Lib thats being inserted by `remix-tests.sol`